docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1911,6 +1911,10 @@ To enable gateway metrics, add spring-boot-starter-actuator as a project depende
 * `httpStatusCode`: The HTTP Status of the request returned to the client.
 * `httpMethod`: The HTTP method used for the request.
 
+In addition, through the property `spring.cloud.gateway.metrics.tags.path.enabled` (by default, set to false), you can activate an extra metric with the tag:
+
+* `path`: Path of the request.
```

Comment:
I have a question and something to note here:

1. Can the value of the `path` have high cardinality?
E.g.: if I have a call to `/api/users/12345` what will be the value of the path? Will it be `/api/users/{id}` or `/api/users/**` or `/api/users/12345`. Is there any way that user/client input leaks into the value of the `path`?

2. Providing an enable/disable mechanism for tags is a slippery slope. The issue with this is that users can configure it and they can end up applications where this is enabled and apps where this is not. These apps can ship metrics to the same backend and aggregating and visualizing metrics where the tags are not aligned is problematic.

I'm not sure this can break metrics backends (maybe it can) but this is "strongly discouraged" by them, e.g. here's Prometheus' opinion about it (label means tag here):
>Metrics with varying label names are very difficult to work with for users and not in line with the guidelines for how to use labels. If label names vary, then you should be using different metric names, as they are fundamentally different metrics.